app/components/finance/npq/payment_overviews/contract_info.rb

```diff
@@ -4,15 +4,17 @@ module Finance
   module NPQ
     module PaymentOverviews
       class ContractInfo < BaseComponent
-        include NPQPaymentsHelper
-
         def initialize(npq_contracts, npq_lead_provider)
           @npq_contracts = npq_contracts
           @npq_lead_provider = npq_lead_provider
         end
 
       private
 
+        def number_to_pounds(number)
```

Comment:
What's the benefit of having multiple copies of this as opposed to have this in a helper that's included where needed?

Reply:
no benefit, more of a liability to be honest. I can introduce a finance helper to share the common stuff